index.js

```diff
@@ -96,7 +96,7 @@ function need(object, path, defaultValue) {
 }
 
 function get(object, path, defaultValue) {
-    return access(object, path, defaultValue, 'debug');
+    return access(object, path, defaultValue);
```

Comment:
Should `access()` check `logType` now in the check for logging? So it can exit that condition early for `get()`